app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -26,17 +26,17 @@
 @(eclDetails: SummaryList,
         organisationDetails: SummaryList,
         contactDetails: SummaryList,
-        isAmendment: Boolean,
+        registrationType: Option[RegistrationType],
         eclReference: Option[String])(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(messages("checkYourAnswers.title")),
-        registrationTypeForTopBanner = if(isAmendment) Some(Amendment) else Some(Initial),
+        registrationTypeForTopBanner = registrationType,
         eclReferenceForStartLink = eclReference
 ) {
     @formHelper(action = CheckYourAnswersController.onSubmit()) {
         <h1 class="govuk-heading-l">@messages("checkYourAnswers.heading")</h1>
 
-        @if(isAmendment) {
+        @if(registrationType.contains(Amendment)) {
```

Comment:
if the function `registrationType.contains(Amendment)` is used a lot throughout the code what you can do is to define an `isAmendment` value inside of RegistrationType, then just use `registrationType.isAmendment`

Reply:
Hmm, contains operates on the Option class and isAmendment would be member of RegistrationType class, that wouldn't work. Or am I missing a point here 😄 ?

Reply:
No that's fair, I didn't see the option - my bad!